src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> reports = new HashMap<>();
```

Comment:
The `reports` map should be private to adhere to the encapsulation principle. Public static fields can be modified from anywhere in the code, which can lead to unpredictable behavior. Consider using a private field and providing access through public methods.